tests/InputFiles/GeneralizedHarmonic/KerrSchild.yaml

```diff
@@ -60,15 +60,6 @@ Amr:
   Verbosity: Quiet
 
 PhaseChangeAndTriggers:
-  - Trigger:
```

Comment:
Why remove this? If you want to remove it, do so in a separate commit since it's unrelated to volume data reading

Reply:
Oh this was just to run the input file without segments, dropped this change